src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -835,6 +835,14 @@ void QgsRasterLayerProperties::apply()
   if ( !mRasterLayer->isValid() )
     return;
 
+  // apply all plugin dialogs
+  const auto constMLayerPropertiesPages = mLayerPropertiesPages;
```

Comment:
Sorry but `auto` is not much appreciated by some core developers, can you use the real type here?